megazords/ios-rust/build-xcframework.sh

```diff
@@ -139,37 +139,22 @@ cp "$WORKING_DIR/DEPENDENCIES.md" "$COMMON/DEPENDENCIES.md"
 
 mkdir -p "$COMMON/Headers"
 
-# First we move the files that are common between both
-# firefox-ios and Focus
+# Library to generate the UniFFI bindings with.  We use an arbitrary target, since that doesn't
```

Comment:
"Arbitrary target" here means that it doesn't matter whether we use `aarch64-apple-ios`, `aarch64-apple-ios-sim`, or `x86_64-apple-ios` (because we build all three, and the metadata for all three should be identical), right?

Reply:
Yes that's exactly right.  In theory, with proc-macros, you could have different definitions depending on the target, but we don't do that yet.